src/rapala.py

```diff
@@ -81,11 +59,22 @@ def init_driver(self) -> webdriver:
         This func initializes the webdriver and disables images
         A wait is initialized with a 5 second timeout
         """
-        options = Options()
-        options.add_argument("--headless")
-        options.add_experimental_option("prefs", self.prefs)
-        service = Service(self.chrome_path)
-        driver = webdriver.Chrome(service=service, options=options)
+    
+        if self.firefox_path:
```

Comment:
We're repeating a lot of code on both sides of your if-block. Can we simplify this since all that differs is the driver called. 

```
driver_dict = {"firefox_driver": webdriver.Firefox, "chrome_driver": webdriver.Chrome}
service = Service(self.driver_path)
driver = driver_dict[driver_type](service=service, options=options)
```

Reply:
This has been attended to, check; _rapala.py_